app/src/org/commcare/activities/FormEntryActivity.java

```diff
@@ -2062,7 +2062,9 @@ public boolean onScroll(MotionEvent e1, MotionEvent e2, float distanceX, float d
 
     @Override
     public void advance() {
-        next();
+        if (!questionsView.isQuestionList() && canNavigateForward()) {
```

Comment:
@phillipm Thanks for flagging. It's worth noting that the issue he was reporting was that the form was getting saved but not submitted, whereas this would just do nothing and wait for you submit as usual. Possible that he doesn't care all that much about the auto-submit, but was correctly reporting odd/buggy behavior. 

Either way, I'm still pretty inclined to say that answering a question should never trigger a form submission, and we should just let Ray know that this is going to change if we think he is relying on it. Do you want to communicate to him since you talk with him often, or should I just drop him a note?